Assets/Scripts/PlayerShooting.cs

```diff
@@ -4,25 +4,43 @@
 
 public class PlayerShooting : MonoBehaviour
 {
+    
     public GameObject arrowPrefab;
-    public float speed = 100f;
+    
+    public static float minForce = 0f;
+    public static float maxForce = 20f; 
+    public static float timeToGetToMaxForce = 2.5f;
+
+    private float shootingStrength = minForce;
+    private float timer = 0f;
 
     // Start is called before the first frame update
     void Start()
     {
-     
+        
     }
 
     // Update is called once per frame
     void Update()
     {
-        if (Input.GetMouseButtonDown(0))
+        if(Input.GetMouseButton(0))
+        {
+            timer += Time.deltaTime;
+            timer = Mathf.Clamp(timer, minForce, timeToGetToMaxForce);
```

Comment:
why is the timer clamped to the minForce that doesn't make sense to me. minForce isn't a unit of time